src/drt/test/drt_aux.py

```diff
@@ -0,0 +1,76 @@
+import drt
+import utl
+
+# NOTE: currently no error checking is done on the inputs as it is
+#       done for the tcl version of detailed_route. If we want to use this
+#       as a basis for a Python api, we will have to add that here
+def detailed_route(design, *,
+                   output_maze="",
+                   output_drc="",
+                   output_cmap="",
+                   output_guide_coverage="",
+                   db_process_node="",
+                   disable_via_gen=False,
+                   droute_end_iter=-1,
+                   via_in_pin_bottom_layer="",
+                   via_in_pin_top_layer="",
+                   or_seed=-1,
+                   or_k=0,
+                   bottom_routing_layer="",
+                   top_routing_layer="",
+                   verbose=1,
+                   param="",
+                   distributed=False,
+                   remote_host=None,
+                   remote_port=None,
+                   shared_volume=None,
+                   cloud_size=None,
+                   clean_patches=False,
+                   no_pin_access=False,
+                   single_step_dr=False,
+                   min_access_points=-1,
+                   save_guide_updates=False):
+
+    router = design.getTritonRoute()
+
+    if param != "":
+        utl.warn(utl.DRT, 8001, "param file overrides all other arguments")
+        router.readParams(param)
+        router.main()
+        return    
```

Comment:
The param file is for legacy support, we can drop it from Python

Reply:
Will do.